zkevm-circuits/src/evm_circuit/util/common_gadget.rs

```diff
@@ -970,13 +970,21 @@ impl<F: Field> CommonErrorGadget<F> {
         opcode: Expression<F>,
         rw_counter_delta: Expression<F>,
     ) -> Self {
-        cb.opcode_lookup(opcode.expr(), 1.expr());
+        Self::construct_with_last_return(cb, opcode, rw_counter_delta, 0.expr(), 0.expr())
+    }
 
-        let rw_counter_end_of_reversion = cb.query_cell();
+    pub(crate) fn construct_with_last_return(
```

Comment:
how about function named as `construct_with_return_data`

Reply:
Rename `construct_with_last_return` to `construct_with_return_data` in commit [66d6d42](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1292/commits/66d6d42bd8a9fe84444c7cd99b561e2175fab448#diff-08d58ebc59ab6c11bf7c049e859441eed2319a03feb3fdffaddaba73f8a8c829L973).